src/components/ProgressData.vue

```diff
@@ -0,0 +1,156 @@
+<template>
```

Comment:
This looks like a quite big component. Maybe, it worth to split it into several smaller ones... 🤔

Reply:
I am going to fix it in the future if the size of the file will grow over 200 lines of code :)